### PR TITLE
[ADD] pos_lot_expiry

### PR DESCRIPTION
**Pull request by etobella**
Evaluate expiry of product in order to check if we can deliver the product

### CODE REVIEW

**Review by FrancoMaxime — APPROVED** [review]
LGTM: code review

File: pos_lot_expiry/__manifest__.py
Comment (cvinh) [inline pos_lot_expiry/__manifest__.py]: I think it should depend on product_expiry as well

Comment (etobella) [reply pos_lot_expiry/__manifest__.py]: Completly right. It was working without it installed, but it has no sense if it is not installed

**Review by cvinh — COMMENTED** [review]
At the end, I would have named the module pos_product_expiry instead of pos_lot_expiry, as expiry is obviously  related to lot by design
and as it depends from product_expiry, the logical seems better to me
WDYT ?

**Comment by etobella** [comment]
Completly true!

File: pos_product_expiry/static/src/js/app/models.esm.js
Comment (cvinh) [inline pos_product_expiry/static/src/js/app/models.esm.js]: a space or a dot is missing before "No changes..."

**Comment by etobella** [comment]
I applied the fixes and it is functional, however, it would be great to make it work better and enable offline mode. To do that, it would be great to integrate it with https://github.com/nguyenminhchien/pos/pull/3

I will wait until it is merged to make it work even better

**Comment by etobella** [comment]
Integrated with pos_lot_selection in order to make it work properly with offline mode.

Also, I added some tests

**Review by cvinh — APPROVED** [review]

**Pull request merged**
